sympy/functions/special/zeta_functions.py

```diff
@@ -384,7 +385,19 @@ def _eval_nseries(self, x, n, logx, cdir=0):
                     s.append(term/k**nu)
                 return Add(*s) + o
 
-        return super(polylog, self)._eval_nseries(x, n, logx, cdir)
+        if z0 == S.One:
+            try:
+                _, exp = z.leadterm(x)
+            except (ValueError, NotImplementedError):
+                return self
+
+            if exp.is_nonnegative:
+                if (nu.is_integer and (1-nu).is_nonpositive) is not True:
+                    self = self.expand()
+                    return gamma(1-nu)*(((log(1/z))**(nu-1))._eval_nseries(x, n, logx, cdir)) + \
+                        super()._eval_nseries(x, n, logx, cdir)
```

Comment:
It's confusing to split the line like this. Just break into actual lines of code:
```python
f = gamma(1-nu)*(((log(1/z))**(nu-1))
f_series = f._eval_nseries(x, n, logx, cdir))
return f_series + super()._eval_nseries(x, n, logx, cdir)
```

Why is this checking `is not True`? What is supposed to happen if the condition is `None`?

Reply:
yep, False should do as even if symbols are passed as parameters they will get stuck in some downstream code during series computations.

Reply:
That check does look odd though. Not something I've seen regularly being used.